package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "origintrail_node",
-    "version": "6.2.0+hotfix.5",
+    "version": "6.2.0+hotfix.6",
```

Comment:
Should we remove hotfix suffix or not yet? 😭

Reply:
I wouldn't remove it, we can do that before going to mainnet :-)